front/src/common/RollingStockSelector/RollingStockCardDetail.tsx

```diff
@@ -103,7 +127,7 @@ export default function RollingStockCardDetail(props) {
               <tr>
                 <td className="text-primary">{t('timetableGamma')}</td>
                 <td>
-                  {data.gamma.value * -1}
+                  {data.gamma?.value ? data.gamma.value * -1 : null}
```

Comment:
```suggestion
                  {data.gamma.value * -1 }
```
Update the openApi.yaml to make the gamma and gamma.value not undefined

Reply:
need to talk about that

Reply:
done